scripts/lint/donotmerge-git-commits.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+DO_NOT_MERGE_KEYWORD="donotmerge"
+DEFAULT_BASE_BRANCH="develop"
+BASE_BRANCH="${CHANGE_TARGET:-$DEFAULT_BASE_BRANCH}"
+FEATURE_BRANCH="${CHANGE_BRANCH:-$(git branch --show-current)}"
+
+if [[ -n "$CI" && "$CI" == "true" ]]; then
+    # Configuring remote origin for CI to resolve origin branches when comparing commits
```

Comment:
A good comment explains "why", and not "what":
```suggestion
    # Default Jenkins refspec does not accept branches other than the one from PR.
```